### DISCUSSION TITLE
Print 'Logs' with format or add Break Lines to '\n' strings

### DISCUSSION THREAD

**Discussion by joshepw** [spatie/ray#578] [Ideas]
Please add support to be able to display a formatted log entry or to allow a string to be displayed when it has line breaks.

<img width="1435" alt="Screen Shot 2021-10-17 at 6 17 02 PM" src="https://user-images.githubusercontent.com/1638280/137650394-8a259cc1-3f09-4510-8443-0244501e92dc.png">

**Comment by axelitus** [comment]
I encountered this today also and would like to see a "formatted" string, particularly line breaks.

**Reply by axelitus** [reply]
Never mind... I just read the docs: https://spatie.be/docs/ray/v1/usage/reference#framework-agnostic-php

What I was looking for is `ray()->text($string)`: _Display the raw text for a string while preserving whitespace formatting._
